presto-docs/src/main/sphinx/release.rst

```diff
@@ -5,6 +5,7 @@ Release Notes
 .. toctree::
     :maxdepth: 1
 
+    release/release-0.283 2023-08-08
```

Comment:
Have you checked that this will be rendered properly @wanglinsong ?

Reply:
updated